src/applications/simple-forms/21-4142/pages/personalInformation2.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ssnUI from 'platform/forms-system/src/js/definitions/ssn';
+import fullSchema from 'vets-json-schema/dist/21-4142-schema.json';
+import { intersection, pick } from 'lodash';
+import { veteranFields } from '../definitions/constants';
+
+const { required, properties } = fullSchema.properties[
+  veteranFields.parentObject
+];
+const pageFields = [veteranFields.ssn, veteranFields.vaFileNumber];
+
+export default {
+  uiSchema: {
+    [veteranFields.parentObject]: {
+      'ui:title': (
+        <h3 className="vads-u-color--gray-dark vads-u-margin-top--0">
+          Identification numbers
+        </h3>
```

Comment:
I know we had a sync about this, but I noticed my sketch has indications like "use an h3, but apply h2 styling", so that may be the answer to some of the questions we had. So maybe some of these need to change to h2 stylings? Might be worth shooting a message to Jeana about, but I know there are several locations on the pages that would need to be update

<img width="866" alt="image" src="https://user-images.githubusercontent.com/53942725/225770918-a83c327e-59ae-4842-93a9-2264cfaa5094.png">